README.md

```diff
@@ -19,7 +19,7 @@ a web project based on [Raxx](https://github.com/CrowdHailer/raxx)/[Ace](https:/
 #### Hello, World!
 ```elixir
 defmodule MyApp do
-  use Ace.HTTP.Service, port: 8080, cleartext: true
+  use Ace.HTTP.Service, port: 8080, cleartext: true, max_line_length: 4096
```

Comment:
I wouldn't put this new option in the default example.
It should be enough to be listed as an option later